src/main/java/com/fabriccommunity/spookytime/block/DeaderBushBlock.java

```diff
@@ -0,0 +1,28 @@
+package com.fabriccommunity.spookytime.block;
+
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.PlantBlock;
+import net.minecraft.entity.EntityContext;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.shape.VoxelShape;
+import net.minecraft.world.BlockView;
+
+public class DeaderBushBlock extends PlantBlock {
+   public static final VoxelShape SHAPE = Block.createCuboidShape(2.0D, 0.0D, 2.0D, 14.0D, 13.0D, 14.0D);
+
+   public DeaderBushBlock(Block.Settings settings) {
+      super(settings);
+   }
+
+   public VoxelShape getOutlineShape(BlockState state, BlockView view, BlockPos pos, EntityContext context) {
```

Comment:
Can we add `@Override` annotations just for clarity? I'll make a quick review comment on every instance of this I see, just so we can keep track of them.